{{cookiecutter.project_name}}/.github/workflows/test_pages_build.yaml

```diff
@@ -0,0 +1,44 @@
+name: Preview cookiecutter.__project_slug}} documentation build
```

Comment:
I think there is a missing `{{` here

Reply:
Good catch @pkalita-lbl! i've made the change to add `{{` in my latest commit.